src/gravel/api/local.py

```diff
@@ -132,7 +135,9 @@ async def get_inventory(
     response_model=NodeStatusReplyModel,
 )
 async def get_status(
-    request: Request, _: Callable = Depends(jwt_auth_scheme)
+    request: Request,
+    jwt: Any = Depends(jwt_auth_scheme),
+    gate: Any = Depends(install_gate),
```

Comment:
Remove this dependency. The API endpoint is called by the installed on the welcome page where the node is not installed right now.